be/src/runtime/global_dict/parser.cpp

```diff
@@ -112,13 +159,48 @@ class DictFuncExpr final : public Expr {
                 return res;
             }
         }
-
-        return nullptr;
     }
 
-    Expr* clone(ObjectPool* pool) const override { return pool->add(new DictFuncExpr(_origin_expr, _dict_opt_ctx)); }
+    ColumnPtr _translate_array(ColumnPtr& array, size_t num_rows) {
```

Comment:
what's the translate here? encode or decode?

Reply:
decode

Reply:
decode, just for keep same name as before